pkgs/tools/misc/owofetch/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "owofetch-rs";
```

The repository name is `owofetch-rs`, but in the `Cargo.toml` the package name is `owofetch`.  This also keeps it consistent with the file path.

```suggestion
  pname = "owofetch";
```